### PR TITLE
Update CHANGELOG.md for 3.2.0

### PR DESCRIPTION
**Pull request by jforissier**
Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>

### CODE REVIEW

**Comment by jforissier** [comment]
As usual, let's gather the `Tested-by:` tags for the next release in this PR. I will create a release candidate tag (-rc1) tomorrow.

**Comment by jforissier** [comment]
Tag 3.2.0-rc1 created, please help test it!

**Comment by vchong** [comment]
First one here ;)

`Tested-by: Victor Chong <victor.chong@linaro.org> (hikey aosp)`.

**Comment by igoropaniuk** [comment]
For 3.2.0-rc1:
```
# xtest -l20
+-----------------------------------------------------
36028 subtests of which 0 failed
75 test cases of which 0 failed
0 test case was skipped
TEE test application done!
```
`Tested-by: Igor Opaniuk <igor.opaniuk@linaro.org> (poplar)`

**Comment by Emantor** [comment]
Tried again with latest master, still running into #2402 on Juno r0.

**Comment by igoropaniuk** [comment]
For 3.2.0-rc1:
```
# xtest -l20
+-----------------------------------------------------
36024 subtests of which 0 failed
74 test cases of which 0 failed
0 test case was skipped
TEE test application done!
```
`Tested-by: Igor Opaniuk <igor.opaniuk@linaro.org> (rpi3)`

**Comment by jbech-linaro** [comment]
> Tested-by: Igor Opaniuk <igor.opaniuk@linaro.org> (rpi3)

FWIW, I missed that Igor already tested this, I've done the same on RPi3 and got the exact same (good) results.
`Tested-by: Joakim Bech <joakim.bech@linaro.org>` (RPi3)

**Comment by lorc** [comment]
For 3.2.0-rc1:

`Tested-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com> (RCAR M3)`

**Comment by JosephChen2017** [comment]
For 3.2.0-rc1:

`Tested-by: Joseph Chen <chenjh@rock-chips.com> (RK322X)`

**Comment by kevin-peng-hao** [comment]
For 3.2.0-rc1

`Tested-by: Kevin Peng <kevinp@marvell.com> (Armada 7K, 8K, 3700)`

**Comment by sahilnxp** [comment]
For 3.2.0-rc1

> +-----------------------------------------------------
> 36024 subtests of which 0 failed
> 74 test cases of which 0 failed
> 0 test case was skipped
> TEE test application done!

`Tested-by: Sahil Malhotra <sahil.malhotra@nxp.com> (NXP - LS1046A-RDB)`

**Comment by jenswi-linaro** [comment]
With https://github.com/OP-TEE/optee_os/pull/2434
Tested-by: Jens Wiklander <jens.wiklander@linaro.org> (Juno)`

**Comment by etienne-lms** [comment]
For 3.2.0-rc1
`Tested-by: Etienne Carriere <etienne.carriere@linaro.org> (stm32mp1)`

With https://github.com/OP-TEE/optee_os/pull/2412
`Tested-by: Etienne Carriere <etienne.carriere@linaro.org> (b2120/b2260)`

Regarding the GP test suite: with https://github.com/OP-TEE/optee_test/pull/294 and https://github.com/OP-TEE/optee_test/pull/292:
`Tested-by: Etienne Carriere <etienne.carriere@linaro.org> (stm32mp1, GP)`
`Tested-by: Etienne Carriere <etienne.carriere@linaro.org> (b2120/b2260, GP)`

**Comment by jenswi-linaro** [comment]
With #2434
`Tested-by: Jens Wiklander <jens.wiklander@linaro.org> (FVP)`

**Comment by jforissier** [comment]
master branch @ af8149de7

`Tested-by: Jerome Forissier <jerome.forissier@linaro.org> (HiKey960 AArch32, Aarch64, GP)`

**Comment by glneo** [comment]
master branch @ af8149d
```
Tested-by: Andrew F. Davis <afd@ti.com> (TI platforms)
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# OP-TEE - version 3.2.0 (2018-06-??)
+
+- Link to the GitHub [release page][github_release_3_2_0].
+- Links to the [commits][github_commits_3_2_0] and
+[pull requests][github_pr_3_2_0] merged into this release.
+
+[github_commits_3_2_0]: https://github.com/OP-TEE/optee_os/compare/3.0.0...3.2.0
```
Comment (jbech-linaro) [inline CHANGELOG.md]: s/3.0.0...3.2.0/3.1.0...3.2.0/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,19 @@
-# OP-TEE - version 3.2.0 (2018-06-??)
+# OP-TEE - version 3.2.0 (2018-07-??)
 
 - Link to the GitHub [release page][github_release_3_2_0].
 - Links to the [commits][github_commits_3_2_0] and
 [pull requests][github_pr_3_2_0] merged into this release.
 
-[github_commits_3_2_0]: https://github.com/OP-TEE/optee_os/compare/3.0.0...3.2.0
+## Knwon issues
```
Comment (jenswi-linaro) [inline CHANGELOG.md]: Known

**Comment by jforissier** [comment]
All: thanks for testing. I have tagged the 3.2.0 release.

**Pull request merged**